pipelines/templates/api-build.yml

```diff
@@ -44,6 +48,7 @@ jobs:
 
       - task: DotNetCoreCLI@2
         displayName: Package Build Artifacts
+        condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
```

Comment:
I don't think we need the condition here??  We need publish command to generate the zip artifact for deployment??  Agree we can save some time by not running unit tests & code coverage at deploy time.